lib/openstudio/alfalfa/creator.rb

```diff
@@ -5,15 +5,20 @@
 require 'sparql/client'
 require 'openstudio'
 
+require_relative 'helpers'
+
 module OpenStudio
   module Alfalfa
     class Creator
       attr_accessor :entities, :model
-      attr_reader :mappings, :templates, :haystack_repo, :brick_repo, :phiot_vocab, :brick_vocab
-
-      # Pass in a model and string, either
-      def initialize(model)
-        @model = model
+      attr_reader :mappings, :templates, :haystack_repo, :brick_repo, :phiot_vocab, :brick_vocab, :metadata_type
+      include OpenStudio::Alfalfa::Helpers
+      ##
+      # @param [String] path_to_model
+      def initialize(path_to_model)
+        m = OpenStudio::Model::Model.load(path_to_model)
```

Comment:
why not just assign this directly to @model, instead of to an intermediate variable which is never used?

Reply:
👍